pkgs/development/node-packages/thelounge-packages-path.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/src/helper.js b/src/helper.js
+index 27352b53..7078e4c5 100644
+--- a/src/helper.js
++++ b/src/helper.js
+@@ -110,6 +110,10 @@ function setHome(newPath) {
+ 	userLogsPath = path.join(homePath, "logs");
+ 	clientCertificatesPath = path.join(homePath, "certificates");
+ 
++	if (process.env.THELOUNGE_PACKAGES !== undefined) {
++		packagesPath = process.env.THELOUNGE_PACKAGES;
++	}
```

Comment:
Is that something we could upstream?

Reply:
It's not, sadly. I had a conversation with an upstream maintainer and they're not interested in even considering something like this until a refactor of how they store data is done.